### PR TITLE
Item manufacturer field

### PR DESCRIPTION
**Pull request by willkrakow**
Added a manufacturer field to Items and made corresponding changes to create/edit forms and item display areas.

Eventually, it would be nice to add enumeration/filtering, similar to what exists for item categories.

### CODE REVIEW

File: src/app/Inventory/Item.tsx [hunk 1]
```diff
@@ -115,20 +117,31 @@ const Item: React.FC<Props> = ({ item }) => {
...
+        </div>
         <div>
           <SelectCreatable
             options={categoryOptions(categories)}
             value={categoryValue}
             onChange={(option: Option<number>) => {
-              update("categoryId", option.value)
-              updateCategory(option.value)
+              update("categoryId", option.value);
+              updateCategory(option.value);
```
Comment (maplethorpej) [inline src/app/Inventory/Item.tsx:144]: Your IDE may be adding semicolons. Not a blocker for this PR, but I typically remove these.

**Review by maplethorpej — APPROVED** [review]

**Comment by maplethorpej** [comment]
Thanks for the contribution! These changes look good.

You'll probably also want to update the pack view repo: https://github.com/Packstack-Tech/packstack-view

🍻

**Comment by nordmanbp** [comment]
First of all, awesome!

Secondly, when are these changes live on the website?

**Comment by willkrakow** [comment]
> Thanks for the contribution! These changes look good.
> 
> 
> 
> You'll probably also want to update the pack view repo: https://github.com/Packstack-Tech/packstack-view
> 
> 
> 
> 🍻

Happy to contribute! Is that pack view repo private? The link 404s for me.

**Comment by maplethorpej** [comment]
Whoops, you're right, it was private. I've set it to public. It's a NextJS project so it's a little different but still typescript/react

**Comment by nordmanbp** [comment]
Working on the inventory page, create pack, and edit pack pages, but not
showing on View Pack pages. Just FYI.

Brett

**Comment by willkrakow** [comment]
Ah good catch. Just made a pull request that adds the field and a column to the packstack-view repo.

**Pull request merged**